kmelia/kmelia-war/src/main/java/org/silverpeas/components/kmelia/servlets/AjaxPublicationsListServlet.java

```diff
@@ -778,17 +763,16 @@ private boolean isAlias(KmeliaSessionController kmeliaScc, PublicationDetail pub
   }
 
   void displayThumbnail(PublicationDetail pub, KmeliaSessionController ksc,
-      SettingBundle publicationSettings, Writer out)
-      throws IOException, NumberFormatException, ThumbnailException {
+      SettingBundle publicationSettings, Writer out) throws IOException, ThumbnailException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.silverpeas.core.io.media.image.thumbnail.ThumbnailException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)